configuration.c

```diff
@@ -4313,7 +4316,7 @@ static void save_keybind_hat(config_file_t *conf, const char *key,
    config[0]        = 'h';
    config[1]        = '\0';
 
-   snprintf(config + 1, sizeof(config) - 1, "%u", hat); 
```

Comment:
Looks like a lot of these changes may have been accidentally committed due to a linter adding spaces to the end of lines. To make it easier to read the diff, can you please undo the extraneous spaces which were added?

Reply:
Done; also rebased to current master